packages/combobox/src/Combobox/Combobox.tsx

```diff
@@ -752,7 +752,7 @@ const Option = <T extends TgphElement>({
   const isVisible =
     !context.searchQuery ||
     doesOptionMatchSearchQuery({
-      label: label?.toString() || children?.toString(),
+      label: label || children,
```

Comment:
should we change this parameter name? Feels like using the word `label` here and within `doesOptionMatchSearchQuery` might be outdated / nondescript now that we pass children in too